component.css

```diff
@@ -0,0 +1,549 @@
+    position: relative;
+    left: -16px;
+    top: -4px;
+}
+.avatar-badge-active{
+    font-size: 10px;
+    background-color:  green;
+    color: white;
+    padding: 1px 6px;
+    border: 2px solid white;
+    border-radius: 50%;
+    position: relative;
+    left: -16px;
+    top: -4px;
+}
+.avatar-text{
+    padding: 11px 18px;
+    border: 1px solid var(--primary);
+    background-color: black;
+    border-radius: 50%;
+    width: fit-content;
+    height: fit-content;
+}
+.avatar-text > span{
+    font-size: 24px;
+    color: yellow;
+}
+                                                /* Badge */
+.badge .icon{
+    width: 30px;
+    height: 30px;
+}
+.badge .count{
+    font-size: 12px;
+    background-color: red;
+    color:white;
+    padding: 2px 5px;
+    border-radius: 5px;
+    position: relative;
+    left: -10px;
+    top:-20px;
+}
+
+                                              /* Button */
+
+.btn{
+    font-size: 18px;
+    padding: 8px 14px;
+    border-radius: 8px;
+    box-shadow: 7px 6px 9px 0px grey;
+    outline: none;
+}
+.btn-primary{
+    color:white;
+    background-color: #007bff;
+    border-color: #007bff;
+}
+.btn-text-primary{
+    color:#007bff;
+    border:1px solid #007bff;
+}
+.btn-secondary {
+    color: white;
+    background-color: #6c757d;
+    border-color: #6c757d;
+}
+.btn-text-secondary{
+    color:#6c757d;
+    border:1px solid #6c757d;
+}
+.btn-success {
+    color: #fff;
+    background-color: #28a745;
+    border-color: #28a745;
+}
+.btn-text-success{
+    color:#28a745;
+    border:1px solid #28a745;
+}
+.btn-danger {
+    color: #fff;
+    background-color: #dc3545;
+    border-color: #dc3545;
+}
+.btn-text-danger{
+    color:#dc3545;
+    border:1px solid #dc3545;
+}
+.btn-warning {
+    color: #212529;
+    background-color: #ffc107;
+    border-color: #ffc107;
+}
+.btn-text-warning{
+    color:#ffc107;
+    border:1px solid #ffc107;
+}
+.btn-link {
+    font-weight: 400;
+    border:none;
+    color: #007bff;
+    background-color: transparent;
+}
+.btn-info {
+    color: #fff;
+    background-color: #17a2b8;
+    border-color: #17a2b8;
+}
+.btn-text-info{
+    color:#17a2b8;
+    border:1px solid #17a2b8;
+}
+.btn-float{
+    font-size: 38px;
+    background-color: red;
+    width: 60px;
+    height: 60px;
+    color: white;
+    border-radius: 50%;
+    outline:none;
+    border: none;
+}
+.primary-badge{
+    padding: 5px 8px;
+    border-radius: 5px;
+    background-color: var(--primary);
+    color: white;
+    font-size: 14px;
+}
+.secondary-badge{
+    padding: 5px 8px;
+    border-radius: 5px;
+    background-color: var(--secondary-hover);
+    color: white;
+    font-size: 14px;
+}
+
+                                                /* Cards */
+
+.card{
+    margin: 20px;
+    box-shadow: 0 0 6px 0px black;
+    width: 300px;
+    height: fit-content;
+}
+.card-dismiss::before{
+    content: "X";
+    position: relative;
+    top: 10px;
+}
+.card-textOnly{
```

Comment:
follow a consistent naming convention, preferrably use dash(-) when dealing with CSS. Here, it should be ".card-text-only"